docs/en/latest/http3.md

```diff
@@ -0,0 +1,109 @@
+---
+title: QUIC/HTTP3 protocol
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+`APISIX` supports HTTP3 protocol. Currently, it only supports opening http3 in `downstream` and `APISIX` connections.
```

Comment:
What is `APISIX` connection?

Reply:
I will rephrase. It actually means "http3 connection between downstream and APISIX"

Reply:
I rephrased it.